조성원/오픈채팅방.js

```diff
@@ -0,0 +1,36 @@
+const solution = (record) => {
+  const answer = [];
+  const users = new Map();
+
+  for (const string of record) {
+    const data = string.split(" ");
```

Comment:
```suggestion
    const [cmd, id, name] = string.split(" ");
```

이런 식으로 구조분해할당해서 사용하면 밑에서 data[인덱스]로 접근하는 것보다 각각 의미하는 게 무엇인지 이해하기 좋을 것 같아요!